### PR TITLE
Handle ps_regs==nullptr case

### PR DESCRIPTION
**Pull request by igor725**

### CODE REVIEW

File: modules/libSceGraphicsDriver/entry.cpp [hunk 1]
```diff
@@ -128,8 +128,24 @@ int SYSV_ABI sceGnmUpdatePsShader(uint32_t* cmdOut, uint64_t size, const uint32_
...
   LOG_USE_MODULE(libSceGraphicsDriver);
   LOG_TRACE(L"%S 0x%08llx", __FUNCTION__, (uint64_t)cmdOut);
-  cmdOut[0] = Pm4::create(size, Pm4::Custom::R_PS_UPDATE);
-  memcpy(&cmdOut[1], ps_regs, 8 + size);
+  if (cmdOut == nullptr || size < 39) return -1;
+  if (ps_regs != nullptr) {
+    cmdOut[0] = Pm4::create(size, Pm4::Custom::R_PS_UPDATE);
+    memcpy(&cmdOut[1], ps_regs, 8 + size);
+  } else {
+    cmdOut[0]  = 0xc0027600;
```
Comment (SysRay) [inline modules/libSceGraphicsDriver/entry.cpp:136]: cmdOut[0] muss Pm4:: ... wie in 133 sein

Comment (igor725) [reply modules/libSceGraphicsDriver/entry.cpp:136]: This code seems to be much higher than any Pm4 command

Comment (SysRay) [reply modules/libSceGraphicsDriver/entry.cpp:136]: Address of the shader is 0x0. It seems to be a reset.
In that case, try:
    cmdOut[0] = Pm4::create(size, Pm4::Custom::R_PS_EMBEDDED);
    cmdOut[1] = 0;

Comment (igor725) [reply modules/libSceGraphicsDriver/entry.cpp:136]: Yeah, it seems to work fine now

**Review by SysRay — APPROVED** [review]

**Pull request merged**